src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj

```diff
@@ -12,6 +12,7 @@
     <Reference Include="Microsoft.AspNetCore.Components.Web" />
     <Reference Include="Microsoft.Extensions.Configuration.Json" />
     <Reference Include="Microsoft.Extensions.Logging" />
+    <Reference Include="Microsoft.Extensions.Logging.Configuration" />
```

Comment:
If we're removing the default `Logging.AddConfiguration` call then hopefully this reference wouldn't be required.